### PR TITLE
Playbook explanation for task files and playbooks w/o tasks

### PR DESCRIPTION
**Pull request by TamiTakamiya**
Modifications of the playbook explanation feature for 
1. Task files, and
2. Playbooks that do not contain tasks, for example, playbooks that have roles only

For 1. We do not enable playbook explanation. The "Explain the current playbook" button on the sidebar will be grayed out.

For 2. We will display an informational message: `Explaining a playbook with no tasks in the playbook is not supported.`

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -296,6 +296,7 @@ export async function activate(context: ExtensionContext): Promise<void> {
         lightSpeedManager,
         pythonInterpreterManager,
       );
+      await lightSpeedManager.lightspeedExplorerProvider.refreshWebView();
```
Comment (TamiTakamiya) [inline src/extension.ts:299]: This was needed for the case that a playbook was not initially recognized as an Ansible file and the user manually set the Ansible language code.  The webview needs to be refreshed at that timing.

Comment (goneri) [reply src/extension.ts:299]: Do we actually need to `await` here?

Comment (TamiTakamiya) [reply src/extension.ts:299]: Maybe not.  Let me try w/o await.

**Review by goneri — APPROVED** [review]
I did the rebase manually and everything works fine. Nice job @TamiTakamiya <3.

File: src/extension.ts (hunk 1 above)
Comment (TamiTakamiya) [reply src/extension.ts:299]: It worked without `await`. I'll get rid of that.

**Review by goneri — APPROVED** [review]

**Comment by goneri** [comment]
Tests are failing because of `TypeError: Cannot read properties of undefined (reading 'openEditor')` which was removed.

**Pull request merged**